README.md

```diff
@@ -1,4 +1,2 @@
-# BT3103
```

Comment:
Can you retain the contents of README as it is?

Reply:
okay, I have made the changes alr!